CHANGELOG.md

```diff
@@ -103,6 +103,15 @@
 * [BUGFIX] Querier: Fix stddev+stdvar aggregations to treat Infinity consistently. #9844
 * [BUGFIX] Ingester: Chunks could have one unnecessary zero byte at the end. #9844
 * [BUGFIX] OTLP receiver: Preserve colons and combine multiple consecutive underscores into one when generating metric names in suffix adding mode (`-distributor.otel-metric-suffixes-enabled`). #10075
+* [BUGFIX] PromQL: Ignore native histograms in `clamp`, `clamp_max` and `clamp_min` functions. #10136
+* [BUGFIX] PromQL: Ignore native histograms in `max`, `min`, `stdvar`, `stddev` functions and instead return an info annotation. #10136
```

Comment:
Are these github numbers correct. Shouldn't it be this issue (#10188)?

Reply:
Yes, they are correct, https://github.com/grafana/mimir/pull/10136 includes https://github.com/prometheus/prometheus/pull/15245 .

Reply:
Okay thanks. Were these just missed from #10136's changelog?

Reply:
Yes, I prioritized getting the code in over going through the changelog at the time - we had plenty to catch up on